machine_management/control_plane_machine_management/cpmso-getting-started.adoc

```diff
@@ -6,37 +6,71 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-For AWS, {product-title} clusters that are created with version 4.12 or later use the Control Plane Machine Set Operator by default. 
+The process for getting started with the Control Plane Machine Set Operator depends on the state of the `ControlPlaneMachineSet` custom resource (CR) in your cluster. 
 
-To use this Operator on AWS clusters that are upgraded to {product-title} version 4.12 or later from an earlier version, you must xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-activating_cpmso-getting-started[activate the Operator].
+Clusters with an active generated CR:: Clusters that have a generated CR with an active state use the Operator by default. No administrator action is required. 
+//+
```

Comment:
Yeah, I initially had this very specific but wanted to move to being more general since granular platform support can be captured (and updated as needed) in the table. But kept them as comments for the sake of my own memory :sweat_smile: